rllib/examples/connectors/connector_v2_nested_observation_spaces.py

```diff
@@ -46,32 +44,11 @@ def _env_to_module_pipeline(env):
         register_env("env", lambda _: CartPoleWithDictObservationSpace())
 
     # Define the AlgorithmConfig used.
-    config = (
+    base_config = (
         get_trainable_cls(args.algo)
         .get_default_config()
-        # Use new API stack for PPO only.
-        .experimental(_enable_new_api_stack=args.enable_new_api_stack)
         .environment("env")
-        .framework(args.framework)
-        .resources(
-            num_gpus=args.num_gpus,  # old stack
-            num_learner_workers=args.num_gpus,  # new stack
-            num_gpus_per_learner_worker=1 if args.num_gpus else 0,
-            num_cpus_for_local_worker=1,
-        )
-        .rollouts(
-            env_to_module_connector=_env_to_module_pipeline,
-            num_rollout_workers=args.num_env_runners,
-            # Set up the correct env-runner to use depending on
-            # old-stack/new-stack and multi-agent settings.
-            env_runner_cls=(
-                None
-                if not args.enable_new_api_stack
-                else SingleAgentEnvRunner
-                if args.num_agents == 0
-                else MultiAgentEnvRunner
-            ),
-        )
+        .rollouts(env_to_module_connector=_env_to_module_pipeline)
```

Comment:
We have to find soon a good way to package model parameters for modules. Passing in `uses_new_env_runners` here should not be done by a user imo but should arrive there by the `AlgorithmConfig` passing it over.

Reply:
Absolutely. We can dissolve this hack as soon as there is no more "hybrid" stack, in which a user uses RLModule (_enable_new_api_stack=True), but is still on RolloutWorker (env_runner_cls=None).
We will do this as soon as we make the new env runners the default when user sets `_enable_new_api_stack=True`. Matter of days from now.